.gitignore

```diff
@@ -57,3 +57,5 @@ out.txt
 # Rails credentials
 config/master.key
 config/credentials.yml.enc
+
+dump.rdb
```

Comment:
Instead of adding `dump.rdb` to `.gitignore`, it would be best for you to simply delete the file / not commit it. Redis should not be outputting anything to the Autolab directory.